packages/next/export/index.js

```diff
@@ -35,6 +36,7 @@ export default async function (dir, options, configuration) {
   const threads = options.threads || Math.max(cpus().length - 1, 1)
   const distDir = join(dir, nextConfig.distDir)
   const subFolders = nextConfig.exportTrailingSlash
+  let apiPage = false
```

Comment:
This should have a more descriptive name (e.g. `warnedAboutApiPage`) or use the `execOnce` helper.